### PR TITLE
Add iAlarm documentation

### PR DESCRIPTION
**Pull request by RyuzakiKK**
**Description:**

Documentation for Antifurto365 iAlarm alarm systems.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#10878

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarm_control_panel.ialarm.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+title: "Antifurto365 iAlarm Control Panel"
+description: "Instructions how to integrate iAlarms alarms into Home Assistant."
+date: 2017-11-30 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: antifurto365-ialarm.png
+ha_category: Alarm
+ha_release: 0.59
```
Comment (armills) [inline source/_components/alarm_control_panel.ialarm.markdown]: This missed the 0.59 merge window, so this needs to be bumped to 0.60.

File: source/_components/alarm_control_panel.ialarm.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+The `ialarm` platform provides connectivity with the Antifurto365 iAlarm alarm systems.
```
Comment (armills) [inline source/_components/alarm_control_panel.ialarm.markdown]: Can we include a link to the product website here?

File: source/_components/alarm_control_panel.ialarm.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+```yaml
+# Example configuration.yaml entry
+alarm_control_panel:
+  - platform: ialarm
+    host: ALARM_SYSTEM_IP
+    username: YOUR_USERNAME
+    password: YOUR_PASSWORD
+```
+
+Configuration variables:
```
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.ialarm.markdown]: Please replace this and use the configuration tag instead:
https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_components/alarm_control_panel.ialarm.markdown (hunk 3 above)
Comment (RyuzakiKK) [reply source/_components/alarm_control_panel.ialarm.markdown]: This should be fixed with aff0d53

**Review by MartinHjelmare — DISMISSED** [review]
Great!

**Review by fabaff — APPROVED** [review]
:bird:

**Comment by wifi75** [comment]
hello it is possible inserte also the log?

**Comment by frenck** [comment]
@wifi75 Please do not comment on old and closed pull requests.

If you are in need of support, please join or Community Forum or Discord chat server. In case of issues, please create a new GitHub issue, so we can start tracking it.

Thanks 👍

**Pull request merged**